docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :release:`0.100.5.dev0 <2019-07-30>`
+* :feature:`4457` Raiden now checks the version of the ethereum node at startup and if it is not supported quits with an appropriate message.
 * :feature:`-` Update WebUI to version 0.9.1 https://github.com/raiden-network/webui/releases/tag/v0.9.1
```

Comment:
Don't we want to add more bugfixes here? E.g. https://github.com/raiden-network/raiden/issues/4377? Or is this only for major bugs?

Reply:
Look at my response to Paul here: https://github.com/raiden-network/raiden/pull/4484#discussion_r308604972